oemof/solph/blocks.py

```diff
@@ -245,62 +247,77 @@ class InvestmentFlow(SimpleBlock):
     **The following sets are created:** (-> see basic sets at
     :class:`.Model` )
 
-    FLOWS
+    INVESTFLOWS
         A set of flows with the attribute :attr:`invest` of type
         :class:`.options.Investment`.
+    CONVEX_INVESTFLOWS
+        A subset of flows with the attribute :attr:`nonconvex` set to `False`.
+    NON_CONVEX_INVESTFLOWS
+        A subset of flows with the attribute :attr:`nonconvex` set to `True`.
     FIXED_FLOWS
-        A set of flow with the attribute :attr:`fixed` set to `True`
+        A set of flow with the attribute :attr:`fixed` set to `True`.
     SUMMED_MAX_FLOWS
```

Comment:
In this logic, it must be SUMMED_MAX_INVESTFLOWS
- actually, one basic question, because I get confused after a while, if there is the 'Investment' Attribute in the flow, there is no `Flow(SimpleBlock)` but an `InvestmentFlow(SimpleBlock)` build, so this is why e.g. the summed_min and summed_max rule need to be in the `InvestmentFlow(SimpleBlock)` class as well, right?

Reply:
replace in code